react/components/QuickOrderAutocomplete.tsx

```diff
@@ -50,7 +50,7 @@ const CustomOption = (props: any) => {
     )
   }
 
-  const buttonClasses = `bn w-100 tl pointer pa4 f6 ${
+  const buttonClasses = `vtex-button bn w-100 tl pointer pa4 f6 ${
```

Comment:
Instead of using `Vtex-button` here, please use CSS Handle to customize. Below is an example of how to add the CSS handle.

import { useCssHandles } from 'vtex.css-handles'

const CSS_HANDLES = [
    '[your handle name]',
] as const

const handles = useCssHandles(CSS_HANDLES)

const buttonClasses = `bn w-100 tl pointer pa4 f6 ${handles.[your handle name]}

Reply:
Done